gym-unity/gym_unity/tests/test_gym.py

```diff
@@ -80,6 +82,33 @@ def test_gym_wrapper_visual(use_uint8):
     assert isinstance(info, dict)
 
 
+@pytest.mark.parametrize("use_uint8", [True, False], ids=["float", "uint8"])
+def test_gym_wrapper_visual_and_vector(use_uint8):
+    mock_env = mock.MagicMock()
+    mock_spec = create_mock_group_spec(
+        number_visual_observations=1, vector_observation_space_size=3
```

Comment:
Please add a test with multiple visual observations to test this line https://github.com/Unity-Technologies/ml-agents/pull/3981/files#diff-ae209b1396be959a453713c25c947969R196

Reply:
Ok! Makes sense